src/platform/unix.rs

```diff
@@ -83,6 +94,13 @@ impl WindowExtUnix for Window {
   fn set_skip_taskbar(&self, skip: bool) {
     self.window.set_skip_taskbar(skip);
   }
+
+  fn new_from_gtk_window(
+    event_loop_window_target: &UnixEventLoopWindowTarget<T>,
+    window: gtk::ApplicationWindow,
+  ) -> UnixWindow {
+    UnixWindow::new_from_gtk_window(event_loop_window_target, window)
+  }
```

Comment:
this shouldn't return a UnixWindow, instead it should be wrapped in the user facing type `Window` and not the platform specific one

Reply:
also this method should return a result

Reply:
Done.